.packit.yaml

```diff
@@ -0,0 +1,21 @@
+---
+specfile_path: python3.spec
+actions:
+  post-upstream-clone:
+  - curl -O https://src.fedoraproject.org/rpms/python3/raw/master/f/python3.spec
+  - curl -O https://src.fedoraproject.org/rpms/python3/raw/master/f/idle3.appdata.xml
+  - curl -O https://src.fedoraproject.org/rpms/python3/raw/master/f/idle3.desktop
+  - curl -O https://src.fedoraproject.org/rpms/python3/raw/master/f/check-pyc-timestamps.py
+  # packit will apply the patches itself
+  - sed '/^Patch/d' -i python3.spec
```

Comment:
We apply some patches bconditionally. How to deal with that?

Reply:
Yes, as I replied in the devel@ thread, your use case is fairly complex and we haven't encountered it yet. We'd need to figure out a technical solution on our side so that it would work well for you and you wouldn't have the need to change the way you work.

https://github.com/packit-service/packit/issues/822